dbms/src/WindowFunctions/tests/gtest_last_value.cpp

```diff
@@ -12,79 +12,34 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <Debug/MockExecutor/WindowBinder.h>
 #include <Interpreters/Context.h>
 #include <TestUtils/ExecutorTestUtils.h>
+#include <TestUtils/WindowTestUtils.h>
 #include <TestUtils/mockExecutor.h>
 
 #include <optional>
 #include <utility>
 
 namespace DB::tests
 {
-// TODO Tests with frame should be added
```

Comment:
Can you refine the comment `// TODO support unsigned int.` in this pr?

Reply:
> 

We will add the support of unsigned int in another pr.